internal/rbd/snapshot.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2020 The Ceph-CSI Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package rbd
+
+import (
+	"context"
+	"strings"
+
+	"github.com/ceph/ceph-csi/internal/util"
```

Comment:
I think it is recommended to place imports of the local project after all others

Reply:
Nope i sent a PR to change order as per the standards.

Reply:
oh, ok, my mistake